src/lib/ServoOutput/devServoOutput.cpp

```diff
@@ -21,30 +22,49 @@ uint16_t servoOutputModeToUs(eServoOutputMode mode)
 {
     switch (mode)
     {
-        case som50Hz: return (1000000U / 50U);
-        case som60Hz: return (1000000U / 60U);
-        case som100Hz: return (1000000U / 100U);
-        case som160Hz: return (1000000U / 160U);
-        case som333Hz: return (1000000U / 333U);
-        case som400Hz: return (1000000U / 400U);
-        default:
-            return 0;
+    case som50Hz:
+        return (1000000U / 50U);
+    case som60Hz:
+        return (1000000U / 60U);
+    case som100Hz:
+        return (1000000U / 100U);
+    case som160Hz:
+        return (1000000U / 160U);
+    case som333Hz:
+        return (1000000U / 333U);
+    case som400Hz:
+        return (1000000U / 400U);
+    case som10KHz:
+        return (1000000U / 10000U);
+    default:
+        return 0;
     }
 }
 
 static void servoWrite(uint8_t ch, uint16_t us)
 {
     const rx_config_pwm_t *chConfig = config.GetPwmChannel(ch);
     if ((eServoOutputMode)chConfig->val.mode == somOnOff)
+    {
         servoMgr->writeDigital(ch, us > 1500U);
+    }
     else
-        servoMgr->writeMicroseconds(ch, us / (chConfig->val.narrow + 1));
+    {
+        if ((eServerPulseWidthMode)chConfig->val.pulseWidthMode == duty)
+        {
+            servoMgr->writeDuty(ch, us - 1000);
```

Comment:
`us` is an `uint16_t` with a min value of `988`.  This may cause overflow and the duty to go max?

Reply:
I will edit the duty caculation to map 988~2012 to 0-1000.

At the very beginning when I write this pr, I mistakenly thought the us range is 1000-2000.

Reply:
I dont think direct mapping of 988~2012 to 0-1000 is correct.  Users with the correct end points 1000~2000 will then miss out on the absolute min and max.

Maybe use constrain() along with mapping 1000~2000 to 0~1000.